lib/ain-cpp-imports/src/lib.rs

```diff
@@ -1,7 +1,7 @@
 use std::error::Error;
 
 #[cfg(not(test))]
-mod bridge;
+pub mod bridge;
```

Comment:
We should keep the bridge module private, but expose the defined TokenAmount struct in the cpp-imports module instead

Reply:
Resolved in [7340314](https://github.com/DeFiCh/ain/pull/2850/commits/7340314660205f1a7c8d0b85a863d404d7975243)